pkg/npm/http-api/example/browser.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
These should probably not be checked in, but *should* be published to NPM. @matildepark